config.template.yml

```diff
@@ -355,12 +355,41 @@ session:
     # Use a unix socket instead
     # host: /var/run/redis/redis.sock
 
-
+    # Optional username to be used with authentication.
+    username: authelia
     # Password can also be set using a secret: https://docs.authelia.com/configuration/secrets.html
     password: authelia
     # This is the Redis DB Index https://redis.io/commands/select (sometimes referred to as database number, DB, etc).
     database_index: 0
 
+  # Settings to connect to Memcache servers
+  # memcache:
+  #  - host: 127.0.0.1
```

Comment:
In my opinion we should include authentication for memcache because we are talking about user sessions here, it's pretty sensitive. Having authentication capabilities is needed in my opinion.